ejenti/pulse_modules/tasksTrigger.py

```diff
@@ -202,10 +201,7 @@ def check_latest_info_json_md5_changed(job_name, platform):
 
         # prepare MD5 folder
         if os.path.exists(md5_folder):
-            if os.path.isdir(md5_folder):
-                shutil.rmtree(md5_folder)
-                os.makedirs(md5_folder)
-            elif os.path.isfile(md5_folder):
+            if os.path.isfile(md5_folder):
                 os.remove(md5_folder)
         else:
             os.makedirs(md5_folder)
```

Comment:
Depends on method functionality, it could `return` after `makedirs` because it's an empty folder and doesn't need to check any change.